pkg/secrets/fetch_secret_test.go

```diff
@@ -144,6 +150,76 @@ func TestExecCommandError(t *testing.T) {
 	assert.Equal(t, "error while running './test/response_too_long/response_too_long"+binExtension+"': command output was too long: exceeded 20 bytes", err.Error())
 }
 
+func TestExecCommandWithHash(t *testing.T) {
+	prevCheckConfigRights := checkConfigRights
+	checkConfigRights = func(string) error { return nil }
+	defer func() {
+		secretBackendCommand = ""
+		secretBackendArguments = []string{}
+		secretBackendTimeout = defaultSecretBackendTimeout
+		secretBackendCommandSHA256 = ""
+		SecretBackendOutputMaxSize = defaultSecretBackendOutputMaxSize
+		checkConfigRights = prevCheckConfigRights
+	}()
+
+	inputPayload := "{\"version\": \"" + PayloadVersion + "\" , \"secrets\": [\"sec1\", \"sec2\"]}"
```

Comment:
Nit: might be easier to use `json.Encode` here.

Reply:
This is a bit of copy&paste from earlier tests, but I agree, I think raw strings could help. I can refactor the older tests separately.